pom.xml

```diff
@@ -160,7 +160,13 @@
 			<artifactId>log4j-core</artifactId>
 			<version>2.17.1</version>
 		</dependency>
-	</dependencies>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-api</artifactId>
+            <version>2.17.2</version>
+            <scope>compile</scope>
+        </dependency>
+    </dependencies>
```

Comment:
as a quick check - this seems to be repeating the log4j import from a few lines above but updating version?